cloud-nio/cloud-nio-impl-drs/src/test/scala/cloud/nio/impl/drs/DrsCloudNioFileProviderSpec.scala

```diff
@@ -25,13 +25,13 @@ class DrsCloudNioFileProviderSpec extends AnyFlatSpecLike with CromwellTimeoutSp
 
   it should "parse a config and create a working file system provider" in {
     val config = ConfigFactory.parseString(
-      """martha.url = "https://from.config"
+      """resolver.url = "https://from.config"
```

Comment:
is there a reason this is `resolver` rather than `drsResolver`?

Reply:
the full path in the config is `filesystems.drs.global.config.resolver.url`, so it seemed a little redundant to have both `drs` and `drsResolver` in the path.

Reply:
Cool, that makes sense! Just wanted to make sure that was intentional in those places :)